pkg/controller/clusterpool/clusterpool_controller.go

```diff
@@ -900,6 +922,35 @@ func (r *ReconcileClusterPool) setAvailableCapacityCondition(pool *hivev1.Cluste
 	return nil
 }
 
+func (r *ReconcileClusterPool) setInventoryValidCondition(pool *hivev1.ClusterPool, valid bool, logger log.FieldLogger) error {
+	status := corev1.ConditionTrue
+	reason := "Valid" // Maybe a different readon?
+	message := "Inventory customization succesfuly applied and reserved"
+	updateConditionCheck := controllerutils.UpdateConditionNever
+	if !valid {
+		status = corev1.ConditionFalse
+		reason = "Invalid"
+		message = "Inventory customization is unvailable or invalid"
+		updateConditionCheck = controllerutils.UpdateConditionIfReasonOrMessageChange
+	}
```

Comment:
How do we know which customization is invalid?

Reply:
Following previous comments regarding `InventoryValid`, the current implementation is irrelevant.